Source/objects.cpp

```diff
@@ -713,52 +713,27 @@ BOOL WallTrapLocOk(int xp, int yp)
 
 void AddL2Torches()
 {
-	int v0;   // esi
-	int v1;   // edi
-	char *v2; // ebx
-	//int v3; // eax
-	int(*v5)[112]; // [esp+Ch] [ebp-Ch]
-	int v6;        // [esp+10h] [ebp-8h]
-	int(*v7)[112]; // [esp+14h] [ebp-4h]
+	int i, j, pn;
 
-	v0 = 0;
-	v7 = dPiece;
-	do {
-		v1 = 0;
-		v2 = &dObject[0][v0 - 1]; /* &dungeon[39][v0 + 39]; */
-		v5 = v7;
-		do {
-			//_LOBYTE(v3) = WallTrapLocOk(v1, v0);
-			if (!WallTrapLocOk(v1, v0))
-				goto LABEL_18;
-			v6 = (*v5)[0];
-			if ((*v5)[0] == 1) {
-				if (random(145, 3))
-					goto LABEL_18;
-				AddObject(OBJ_TORCHL2, v1, v0);
-			}
-			if (v6 == 5) {
-				if (random(145, 3))
-					goto LABEL_18;
-				AddObject(OBJ_TORCHR2, v1, v0);
-			}
-			if (v6 == 37) {
-				if (random(145, 10) || *(v2 - 111))
-					goto LABEL_18;
-				AddObject(OBJ_TORCHL, v1 - 1, v0);
-			}
-			if (v6 == 41) {
-				if (!random(145, 10) && !*v2)
-					AddObject(OBJ_TORCHR, v1, v0 - 1);
-			}
-		LABEL_18:
-			++v5;
-			++v1;
-			v2 += 112;
-		} while (v1 < 112);
-		v7 = (int(*)[112])((char *)v7 + 4);
-		++v0;
-	} while ((signed int)v7 < (signed int)dPiece[1]);
+	for (i = 0; i < MAXDUNY; i++) {

```

Comment:
Usually `j` is mapped to `y`

Reply:
Alright, I thought of putting `i`, `j` according to order for the loops to also emphasize inefficiency of such iteration order but I agree that it's more intuitive to have them correspond to `x`, `y`